src/data/matrix/basic.lean

```diff
@@ -891,7 +953,7 @@ instance : star_ring (matrix n n α) :=
   star_add := λ M N, by { ext, simp, },
   star_mul := λ M N, by { ext, simp [mul_apply], }, }
```

Comment:
You should be able to replace these proofs with your new lemmas above

Reply:
I think the odd ones look fine as they are already very simple. Using lemmas above will make the line longer.

Reply:
Line length isn't the only thing that matters - in this case, it's valuable to be able to tell lean "look, we already proved this'. The proof will run faster too.

Reply:
things are now done, Eric.